docs/zh_cn/advanced_tutorials/registry.md

```diff
@@ -119,7 +120,7 @@ def build_activation(cfg, registry, *args, **kwargs):
 并将 `build_activation` 传递给 `build_func` 参数
 
 ```python
-ACTIVATION = Registry('activation', build_func=build_activation, scope='mmengine')
+ACTIVATION = Registry('activation', build_func=build_activation, scope='mmengine', locations=['mmengine.models.activations'])
```

Comment:
Users may be confused by this value `locations=['mmengine.models.activations']` because this example can not be executed as expected.

Reply:
This PR can be merged after refining the docs.